src/Simplex/Chat/Controller.hs

```diff
@@ -736,7 +736,7 @@ data ChatResponse
   | CRNetworkStatuses {user_ :: Maybe User, networkStatuses :: [ConnNetworkStatus]}
   | CRHostConnected {protocol :: AProtocolType, transportHost :: TransportHost}
   | CRHostDisconnected {protocol :: AProtocolType, transportHost :: TransportHost}
-  | CRGroupInvitation {user :: User, groupInfo :: GroupInfo}
+  | CRGroupInvitation {user :: User, groupId :: GroupId, groupName :: GroupName}
```

Comment:
I think this particular event is used in other contexts? so it may need full group info?

Reply:
no